README.md

```diff
@@ -91,17 +96,17 @@ plugin.get_deployment(name=<deployment name>)
 ```
 
 ### Run Prediction on deployed model
-Predict API enables to run prediction on the deployed model. 
+Predict API enables to run prediction on the deployed model.
 
-CLI takes json file path as input. However, input to the python plugin can be one among the three types
-DataFrame, Tensor or Json String.
+CLI takes the json file path as input. However, input to the python plugin can be one among the three types
```

Comment:
Please reword to "For the prediction inputs, DataFrme, Tensor and Json formats are supported. The python API supports all of these three formats. When invoked via command line, one needs to pass the json file path that contains the inputs".

Reply:
Have incorporated the suggested change